app/assets/javascripts/discourse/app/components/topic-status.gjs

```diff
@@ -1,30 +1,45 @@
 import Component from "@glimmer/component";
-import { hash } from "@ember/helper";
+import { concat, get, hash } from "@ember/helper";
 import { on } from "@ember/modifier";
 import { action } from "@ember/object";
 import { service } from "@ember/service";
 import { and } from "truth-helpers";
 import PluginOutlet from "discourse/components/plugin-outlet";
 import icon from "discourse/helpers/d-icon";
+import element from "discourse/helpers/element";
+import TopicStatusIcons from "discourse/helpers/topic-status-icons";
 import { i18n } from "discourse-i18n";
 
 export default class TopicStatus extends Component {
   @service currentUser;
+  @service site;
+
+  get wrapperElement() {
+    return element(this.args.tagName ?? "span");
+  }
 
   get canAct() {
+    // TODO: @disableActions -> !@interactive
```

Comment:
Is this something that should be done before merging? Or just a reminder for later?

Reply:
I'll be updating this double-negative argument (`@disableActions={{false}}`, bleh :P) in another PR